cluster/eval_server/add_model.py

```diff
@@ -0,0 +1,155 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+sys.path.insert(0, '.')
+
+import datetime
+import hashlib
+import os
+import shutil
+
+from absl import flags
+from google.cloud import bigtable
+from tensorflow import gfile
+import fire
+
+from bigtable_input import METADATA
+
+
+flags.mark_flags_as_required([
+    "cbt_project", "cbt_instance", "cbt_table"
+])
+
+FLAGS = flags.FLAGS
+
+
+# TODO(sethtroisi): Unify with oneoffs
+MODEL_ROW = "m_{}_{:0>10}"
```

Comment:
nit: give example in comment.

Reply:
This get's automatically generated as m_eval_{HASH} see line ~104